samples/dfu/ab/README.rst

```diff
@@ -0,0 +1,105 @@
+
+.. contents::
+   :local:
+   :depth: 2
+
+The sample demonstrates how to configure the application, so it will be updated using the "A/B" method using MCUBoot as well as a sample way to perform a device healthcheck before confirming the image after the update.
+
+You can update the sample using SMP protocol with UART or Bluetooth® Low Energy.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-sample-yaml::
+
+You need the nRF Device Manager app for SUIT update over Bluetooth Low Energy:
+
+* `nRF Device Manager mobile app for Android`_
+* `nRF Device Manager mobile app for iOS`_
+
+
+Overview
+********
+
+This sample demonstrates firmware update using "A/B" method.
+This method allows to have two copies of the application in the NVM memory.
+It is possible to switch between these copies, without the need to perform swap what significantly reduces time of device's unavailability during the update.
+Switch between images can be triggered by application or for example HW button.
+
+This sample application implements a Simple Management Protocol (SMP) server.
+SMP is a basic transfer encoding for use with the MCUmgr management protocol.
+For more information about MCUmgr and SMP, please see :ref:`device_mgmt`.
+
+This sample application supports the following MCUmgr transports by default:
+
+    * Bluetooth
+    * Serial (UART)
+
+
+Configuration
+*************
+
+The :kconfig:option:`CONFIG_N_BLINKS` Kconfig option sets the number of times the LED corresponding to the currently active slot blinks (LED0 for slot A, LED1 for slot B).
+The default value of the option is set to ``1``, causing a single blink to indicate *Version 1*.
+You can increment this value to represent an update, such as set it to ``2`` to indicate *Version 2*.
+
+Use :kconfig:option:`CONFIG_APP_HEALTH_CHECK_FAILURE` Kconfig option to emulate a "broken" application that does not pass the self-test.
+
+To ensure the updated build is preferred after a DFU, set the :kconfig:option:`CONFIG_MCUBOOT_IMGTOOL_SIGN_VERSION` option to a higher version number then currently running on the device.
+Failing to do so may result in the image not being selected on the next reboot, depending on the slot used for the update.
+
+User interface
+**************
+
+.. tabs::
+
+   .. group-tab:: nRF54 DKs
+
+      LED 0:
+         This LED indicates that the application is running from slot A.
+         This LED is controlled as "active low", meaning it will turn on once the application is booted and blinks (turns off) in short intervals.
+         The number of short blinks is configurable using the :kconfig:option:`CONFIG_N_BLINKS` Kconfig option.
+         It will be off if the application is running from slot B.
+
+      LED 1:
+         This LED indicates that the application is running from slot B.
+         This LED is controlled as "active low", meaning it will turn on once the application is booted and blinks (turns off) in short intervals.
```

Comment:
```suggestion
         It is controlled as "active low", meaning it will turn on once the application is booted and blinks (turns off) in short intervals.
```